app/controllers/api/applications_controller.rb

```diff
@@ -3,7 +3,7 @@ class Api::ApplicationsController < Api::ApiApplicationController
   load_and_authorize_resource :application
 
   def index
-    render status: 200, json: @applications
+    render status: 200, json: @applications.lti
```

Comment:
`@applications` would be an array of activerecord models. So what is `.lti` doing here?